packages/odyssey/src/scss/abstracts/_fonts.scss

```diff
@@ -13,28 +13,55 @@
 // limitations under the License.
 
 @font-face {
-  font-family: 'Proxima Nova';
+  font-family: 'Whyte';
```

Comment:
I know we're handicapped by our ability to check-in our font files, so I wonder if we should move this abstract out of Odyssey. To me, it's a bit weird to apply markup based on an assumption that the consuming client has these font files under a specific path.

Is this a common convention for SCSS libraries when you cannot embed licensed files?